docs/transform/modernize-userinterface-site-pages-api.md

```diff
@@ -523,9 +523,9 @@ Name|Description
 
 #### Input parameters
 
-Name|Description
-:-----|:----------
-{ContactLoginName}|User account to lookup (in i:0#.f|membership|joe@contoso.onmicrosoft.com format)
+Name|Description| Membership
```

Comment:
Let's keep this table to two columns, and expose the pipes in cells like this:
|Name|Description|
|:---|:---|
|{ContactLoginName}|User account to lookup (in i:0#.f\|membership\|joe@contoso.onmicrosoft.com format)|